freeipa/src/main/java/com/sequenceiq/freeipa/service/upgrade/UpgradeValidationService.java

```diff
@@ -22,16 +22,18 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(UpgradeValidationService.class);
 
+    private static final int MAX_NUMBER_OF_INSTANCES_FOR_UPGRADE = 3;
+
     @Inject
     private EntitlementService entitlementService;
 
     public void validateStackForUpgrade(Set<InstanceMetaData> allInstances, Stack stack) {
         if (allInstances.isEmpty()) {
             LOGGER.warn("Instances are empty for stack.");
             throw new BadRequestException("There are no instances available for upgrade!");
-        } else if (allInstances.size() > 2) {
+        } else if (allInstances.size() > MAX_NUMBER_OF_INSTANCES_FOR_UPGRADE) {
```

Comment:
they can't create actually. in case of light duty we create 2 instances, in case of medium we create it with 3 instances. There is a validation in place which would prevent creating freeipa with more than 3 instances